lua/dap/session.lua

```diff
@@ -314,7 +320,7 @@ local function with_win(win, fn, ...)
   local cur_win = api.nvim_get_current_win()
   api.nvim_set_current_win(win)
   local ok, err = pcall(fn, ...)
-  api.nvim_set_current_win(cur_win)
+  pcall(api.nvim_set_current_win, cur_win)
```

Comment:
This change is to avoid errors when the original window closes while the function runs. It can happen with dapui when using a floating window which will close when a window switches